src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3425,6 +3566,25 @@ enum StopAlertType {
   TRIPS
 }
 
+"Whether stop has been updated through a realtime update and if so, how."
+enum StopRealTimeState {
+  "The stop has been added through a realtime update."
+  ADDED
+  "The stop has been canceled through a realtime update."
+  CANCELED
+  "The realtime feed has indicated that there is no data available for this stop."
+  NO_DATA
+  """
+  The vehicle has arrived to the stop or already visited it and the times are no longer estimates.
+  Note, not all realtime feeds indicate this information even if the vehicle has already passed the stop.
+  """
+  RECORDED
+  "There have been no realtime updates."
+  UNUPDATED
```

Comment:
Unupdated? Don't we call these stops times "scheduled"? We could also use "NOT_UPDATED".

Reply:
We have called them that but since this enum was about realtime information, the meaning of scheduled is not completely clear. In GTFS RT terminology, SCHEDULED means "This trip is running according to a GTFS schedule, or is close enough to still be associated with it." which also allows realtime updated times. I thought about NOT_UPDATED but I thought UNUPDATED was the proper english version of it :D.

Ps. I'm still in progress of refactoring this enum so it can be used in the future to portray planned cancellations perhaps.

Reply:
I decided to remove this and some other stuff from the new stop time type since I don't need them and don't want to design them now.

We can figure out what we actually want when there is a need for it. We discussed this in a dev meeting and had some concerns about what name to use for this UNUPDATED (we discussed about SCHEDULED and PLANNED). Also, a stop might be added and updated later, therefore those two should not be in the same enum.

Reply:
It is unfortunately a NETEX only feature currently. There is uncertainty in GTFS RT Trip Updates which in some cases might mean this but not always, so we can't really use it to populate this information.